test/extended/networking/livemigration.go

```diff
@@ -537,6 +570,75 @@ func verifyVMMAC(virtClient *kubevirt.Client, vmName, expectedMAC string) {
 		Should(Equal(expectedMAC))
 }
 
+func duplicateVM(cli *kubevirt.Client, vmNamespace, vmName string) {
+	GinkgoHelper()
+	duplicateVMName := vmName + "-duplicate"
+	By(fmt.Sprintf("Duplicating VM %s/%s to %s/%s", vmNamespace, vmName, vmNamespace, duplicateVMName))
+
+	vmiSpecJSON, err := cli.GetJSONPath("vmi", vmName, "{.spec}")
+	Expect(err).NotTo(HaveOccurred())
+	var vmiSpec map[string]interface{}
+	Expect(json.Unmarshal([]byte(vmiSpecJSON), &vmiSpec)).To(Succeed())
+
+	originalVMIRawAnnotations, err := cli.GetJSONPath("vmi", vmName, "{.metadata.annotations}")
+	Expect(err).NotTo(HaveOccurred())
+
+	originalVMIAnnotations := map[string]string{}
+	Expect(json.Unmarshal([]byte(originalVMIRawAnnotations), &originalVMIAnnotations)).To(Succeed())
+
+	var vmiCreationOptions []kubevirt.Option
+	var vmiExpectations []func()
+	if requestedIPs, hasIPRequests := originalVMIAnnotations[kvIPRequestsAnnot]; hasIPRequests {
+		vmiCreationOptions = append(
+			vmiCreationOptions,
+			kubevirt.WithAnnotations(ipRequests(requestedIPs)),
+		)
+		vmiExpectations = append(vmiExpectations, func() {
+			waitForVMPodEventWithMessage(
+				cli,
+				vmNamespace,
+				duplicateVMName,
+				"IP is already allocated",
+				2*time.Minute,
+			)
+		})
+	} else if hasMACAddressRequest(cli, vmName) {
```

Comment:
don't we want to drop the "else" ? Otherwise, how will we be able to create a VM w/ MAC & IP requests ? 

As this stands, I think it'll get either MAC **or** (exclusive or) IP.

Reply:
I removed the MAC related code. We will reintroduce it when we add the MAC test.
Wanted to again clarify though - the pod will not err on both MAC and IP conflicts, it will return err on the first problem it finds and rollover.

Reply:
hmm let's pick up this thread in the follow up PR where we add the MAC dupliacate